[8.13][cloud] CNVM Findings page enhancements (#4937)
elastic/security-docs: docs/cloud-native-security/vuln-management-findings.asciidoc

[[vuln-management-findings]]
= Findings page

> The Vulnerabilities tab of the Findings page displays the vulnerabilities detected by the <<vuln-management-overview, CNVM integration>>. 

Reviewer: ```suggestion
The **Vulnerabilities** tab of the Findings page displays the vulnerabilities detected by the <<vuln-management-overview, CNVM integration>>. 
```

image::images/cnvm-findings-page.png[The Vulnerabilities tab of the Findings page]

[discrete]
[[cnvm-what-are-findings]]
== What are CNVM findings?

CNVM findings represent security vulnerabilities detected in your cloud. They include metadata such as the CVE identifier, CVSS score, severity, affected package, and fix version if available, as well as information about impacted systems.

Clicking on a finding provides a detailed description of the vulnerability, and any available remediation information.

[discrete]
[[vuln-findings-grouping]]
== Group and filter findings

To help you prioritize remediation efforts, you can organize findings in various ways.

[discrete]
=== Group findings

. Click **Group vulnerabilities by:** to open a list of fields by which you can group your data. Select one of the suggested fields, or click **Custom field** to choose your own. You can select up to three group-by fields at once. 
. When grouping is turned on, click a group to expand it and examine all sub-groups or findings within that group.
. To turn off grouping, click **Group findings by:** and select **None**.

> NOTE: Multiple groupings apply to your data in the order you selected them; for example if you first select **Cloud account**, and then select **Resource**, the top-level grouping will be based on **Cloud account**, and its subordinate grouping will be based on **Resource**, as in the following screenshot:

Reviewer: Minor tweak to the last sentence for clarity.

```suggestion
NOTE: Multiple groupings apply to your data in the order you selected them. For example, if you first select **Cloud account**, and then select **Resource**, the top-level grouping will be based on **Cloud account**, and its subordinate grouping will be based on **Resource**, as demonstrated in the following screenshot:
```

image::images/cnvm-findings-grouped.png[The Vulnerabilities tab of the Findings page]

[discrete]
[[cnvm-filter-findings]]
=== Filter findings
You can filter the data in two ways:

> * *The KQL search bar*: For example, search for `vulnerability.severity : "HIGH"` to view high severity vulnerabilities.

Reviewer 2: ```suggestion
* *KQL search bar*: For example, search for `vulnerability.severity : "HIGH"` to view high severity vulnerabilities.
```

> * *In-table value filters*: Hover over a finding to display available inline actions. Use the Filter In (plus) and Filter Out (minus) buttons.

Reviewer: ```suggestion
* *In-table value filters*: Hover over a finding to display available inline actions. Use the **Filter In** (plus) and **Filter Out** (minus) buttons.
```

[discrete]
[[cnvm-customize-the-findings-table]]
=== Customize the Findings table

> When grouping is disabled, you can use the toolbar buttons in the upper-left of the Findings table to select which columns appear:

Reviewer 3: same consideration of grouping not need to be disabled

> When grouping is disabled, you can use the toolbar buttons in the upper-left of the Findings table to select which columns appear:

Reviewer: ```suggestion
When grouping is turned off, you can use the toolbar buttons in the upper-left of the Findings table to select which columns appear:
```

* **Columns**: Select the left-to-right order in which columns appear.

> * **Sort fields** or **_n_ fields sorted**: Sort the table by one or more columns, or turn sorting off.

Reviewer 4: ```suggestion
* **Sort fields**: Sort the table by one or more columns, or turn sorting off.
```
Looking at the updated screenshots, it looks like this copy has changed and it says **Sort fields _n_** even when sorting is on.
